### PR TITLE
Added Russian Translations

### PR DESCRIPTION
**Pull request by CriticalFlaw**

### CODE REVIEW

File: config/mastercomfig/cfg/lang/ru.cfg [hunk 1]
```diff
@@ -0,0 +1,18 @@
+alias flat_mouse_echo"echo Примён модуль, Плоская Мышь"
```
Comment (h9lk) [inline config/mastercomfig/cfg/lang/ru.cfg]: "Применён модуль", it's also not necessary translate module name

File: config/mastercomfig/cfg/lang/ru.cfg [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+alias low_mem_echo"echo Примён модуль, Мало Памяти"
```
Comment (h9lk) [inline config/mastercomfig/cfg/lang/ru.cfg]: "Применён модуль", here it's also better leave module name "low memory"

File: config/mastercomfig/cfg/lang/ru.cfg [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+alias low_mem_warn"echo ВНИМАНИЕ: с этим модулом ваща игра будет играть ХУДШЕ если у вас много компюторной памяти!!!"
```
Comment (h9lk) [inline config/mastercomfig/cfg/lang/ru.cfg]: "С этим модулем ваша игра будет выглядеть хуже, если у вас больше озу"

File: config/mastercomfig/cfg/lang/ru.cfg [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+alias no_tutorial_echo"echo Примён модуль, Отключить Тренировку"
+alias null_movement_echo"echo Примён модуль, Null-Canceling Движения"
+alias transparent_vm_echo"echo Примён модуль, Прозрачные Модели Оружия"
+alias ultra_preset_echo"echo Выбрана Ультра настройка"
+alias high_preset_echo"echo Выбрана Высокая настройка"
+alias medium_high_preset_echo"echo Выбрана Средня-Высокая настройка"
+alias medium_preset_echo"echo Выбрана Средния настройка"
+alias medium_low_preset_echo"echo Выбрана Средния-Ниская настройка"
```
Comment (h9lk) [inline config/mastercomfig/cfg/lang/ru.cfg]: "Низкая"

File: config/mastercomfig/cfg/lang/ru.cfg [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+alias low_preset_echo"echo Выбрана Ниская настройка"
```
Comment (h9lk) [inline config/mastercomfig/cfg/lang/ru.cfg]: "Низкая"

File: config/mastercomfig/cfg/lang/ru.cfg [hunk 6]
```diff
@@ -0,0 +1,18 @@
+alias flat_mouse_echo"echo Примён модуль, Плоская Мышь"
+alias low_mem_echo"echo Примён модуль, Мало Памяти"
+alias low_mem_warn"echo ВНИМАНИЕ: с этим модулом ваща игра будет играть ХУДШЕ если у вас много компюторной памяти!!!"
+alias no_tutorial_echo"echo Примён модуль, Отключить Тренировку"
```
Comment (h9lk) [inline config/mastercomfig/cfg/lang/ru.cfg]: "Применён модуль - Отключение тренировки"

File: config/mastercomfig/cfg/lang/ru.cfg [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+alias null_movement_echo"echo Примён модуль, Null-Canceling Движения"
```
Comment (h9lk) [inline config/mastercomfig/cfg/lang/ru.cfg]: "Применён", if you want translate names then "нулевая отмена движения"

File: config/mastercomfig/cfg/lang/ru.cfg [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+alias transparent_vm_echo"echo Примён модуль, Прозрачные Модели Оружия"
+alias ultra_preset_echo"echo Выбрана Ультра настройка"
+alias high_preset_echo"echo Выбрана Высокая настройка"
+alias medium_high_preset_echo"echo Выбрана Средня-Высокая настройка"
+alias medium_preset_echo"echo Выбрана Средния настройка"
+alias medium_low_preset_echo"echo Выбрана Средния-Ниская настройка"
+alias low_preset_echo"echo Выбрана Ниская настройка"
+alias very_low_preset_echo"echo Выбрана Очень Ниская настройка"
```
Comment (h9lk) [inline config/mastercomfig/cfg/lang/ru.cfg]: "Низкая"

File: config/mastercomfig/cfg/lang/ru.cfg [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+alias none_preset_echo"echo Настройка была не выбрана"
+alias version_comfig"echo мастеркомм версия: 9.6.0 | Июнь 28, 2021"
+alias block_antialias_echo"echo Эта команда отключена чтобы предотвратить перезагрузку системы материалов. Чтобы это поменять, включите модуль Сглаживание."
+alias mm_override_echo"echo Вычистели настройки мастеркомм для пользы в соревновательной игре"
+alias modules_echo"echo Модуль примён"
```
Comment (h9lk) [inline config/mastercomfig/cfg/lang/ru.cfg]: "Применён"

File: config/mastercomfig/cfg/lang/ru.cfg [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+alias high_preset_echo"echo Выбрана Высокая настройка"
+alias medium_high_preset_echo"echo Выбрана Средня-Высокая настройка"
+alias medium_preset_echo"echo Выбрана Средния настройка"
+alias medium_low_preset_echo"echo Выбрана Средния-Ниская настройка"
+alias low_preset_echo"echo Выбрана Ниская настройка"
+alias very_low_preset_echo"echo Выбрана Очень Ниская настройка"
+alias none_preset_echo"echo Настройка была не выбрана"
+alias version_comfig"echo мастеркомм версия: 9.6.0 | Июнь 28, 2021"
+alias block_antialias_echo"echo Эта команда отключена чтобы предотвратить перезагрузку системы материалов. Чтобы это поменять, включите модуль Сглаживание."
+alias mm_override_echo"echo Вычистели настройки мастеркомм для пользы в соревновательной игре"
```
Comment (h9lk) [inline config/mastercomfig/cfg/lang/ru.cfg]: "Очищение настроек"

File: config/mastercomfig/cfg/lang/ru.cfg [hunk 11]
```diff
@@ -0,0 +1,18 @@
+alias flat_mouse_echo"echo Примён модуль, Плоская Мышь"
+alias low_mem_echo"echo Примён модуль, Мало Памяти"
+alias low_mem_warn"echo ВНИМАНИЕ: с этим модулом ваща игра будет играть ХУДШЕ если у вас много компюторной памяти!!!"
+alias no_tutorial_echo"echo Примён модуль, Отключить Тренировку"
+alias null_movement_echo"echo Примён модуль, Null-Canceling Движения"
+alias transparent_vm_echo"echo Примён модуль, Прозрачные Модели Оружия"
```
Comment (h9lk) [inline config/mastercomfig/cfg/lang/ru.cfg]: "Применён модуль - модели прозрачного оружия"

File: config/mastercomfig/cfg/lang/ru.cfg [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+alias ultra_preset_echo"echo Выбрана Ультра настройка"
+alias high_preset_echo"echo Выбрана Высокая настройка"
+alias medium_high_preset_echo"echo Выбрана Средня-Высокая настройка"
+alias medium_preset_echo"echo Выбрана Средния настройка"
+alias medium_low_preset_echo"echo Выбрана Средния-Ниская настройка"
+alias low_preset_echo"echo Выбрана Ниская настройка"
+alias very_low_preset_echo"echo Выбрана Очень Ниская настройка"
+alias none_preset_echo"echo Настройка была не выбрана"
```
Comment (h9lk) [inline config/mastercomfig/cfg/lang/ru.cfg]: "Настройка не была выбрана"

**Review by h9lk — COMMENTED** [review]
Standard mistakes when translating with translator

File: config/mastercomfig/cfg/lang/ru.cfg (hunk 6 above)
Comment (CriticalFlaw) [reply config/mastercomfig/cfg/lang/ru.cfg]: Don't think mine was necessarily wrong here, but in any case it was changes to English to match other module lines. Thanks!

File: config/mastercomfig/cfg/lang/ru.cfg (hunk 7 above)
Comment (CriticalFlaw) [reply config/mastercomfig/cfg/lang/ru.cfg]: Changed the module name to English to match others.

**Comment by CriticalFlaw** [comment]
> Standard mistakes when translating with translator

Thank you for correcting my mistakes. My 5th grade Russian can only get me so far. 😋

**Comment by micwoj92** [comment]
Why are there like 5 open prs on Russian translation? I get that the Union collapsed, but come on guys, try to coordinate!

**Comment by CriticalFlaw** [comment]
> Why are there like 5 open prs on Russian translation? I get that the Union collapsed, but come on guys, try to coordinate!

Why would that bother you? Winth is doing a good job with translating the documentation.

**Comment by micwoj92** [comment]
My mailbox is getting spammed!

**Comment by h9lk** [comment]
> 
> 
> My mailbox is getting spammed!

write that your mail is spammed - you were not too lazy, but how to make folder in mail for github letters, make letter filter there and mark "as read" - no, too easy? 
img

**Comment by micwoj92** [comment]
What email client are you using?

**Comment by h9lk** [comment]
> 
> 
> What email client are you using?

yandex, if u want i can do this in gmail

img

**Comment by micwoj92** [comment]
Thank you.

**Comment by CriticalFlaw** [comment]
So could this get reviewed, please?

Are there additional changes that need to be made?

**Review by mastercoms — APPROVED** [review]

**Pull request merged**